examples/atlas-datalake-pipeline/main.tf

```
@@ -0,0 +1,46 @@
+resource "mongodbatlas_project" "atlas-project" {
+  org_id = var.atlas_org_id
+  name   = var.atlas_project_name
+}
+
+resource "mongodbatlas_cluster" "automated_backup_test" {
+  project_id                  = mongodbatlas_project.atlas-project.id
+  name                        = var.cluster_name
+  provider_name               = "GCP"
+  provider_region_name        = "US_EAST_4"
+  provider_instance_size_name = "M30"
+  cloud_backup                = true # enable cloud backup snapshots
+  mongo_db_major_version      = "4.4"
```

```suggestion
  mongo_db_major_version      = "6.0"
```
Suggest let's use latest default MongoDB version in examples going forward